docs: add helper scripts install for proxmox (#197)
stonith404/pocket-id: docs/docs/setup/installation.md

You can now sign in with the admin account on `http://localhost/login/setup`.

> ### Install on Proxmox using Helper Scripts

Reviewer: ```suggestion
### Proxmox
```

I would keep the title simple

> Run the following script as root in your proxmox shell. 
> 
> See [Here](https://community-scripts.github.io/ProxmoxVE/scripts?id=pocketid) for more information.

Reviewer: ```suggestion
Run the [helper script](https://community-scripts.github.io/ProxmoxVE/scripts?id=pocketid) as root in your Proxmox shell. 
```
